src/app/containers/StakePage/components/HistoryEventsTable.tsx

```diff
@@ -51,80 +47,18 @@ export function HistoryEventsTable() {
 
   useEffect(() => {
     async function getHistory() {
-      let genesis: void, team: void, origin: void;
-      const stake = await eventReader
-        .getPastEvents('staking', 'TokensStaked', {
-          staker: account,
-        })
-        .then(res => {
-          setEventsHistory(
-            (res as any).sort(
-              (x, y) =>
-                new Date(y.eventDate).getTime() -
-                new Date(x.eventDate).getTime(),
-            ),
-          );
+      axios
+        .get(backendUrl[chainId] + '/events/stake/' + account + '')
```

Comment:
You don't need `+ ''`, it's fully functional without it.

Also, you can use string interpolation and rewrite it to `${backendUrl[chainId]}/events/stake/${account}`, I think it's better for readability.

Reply:
fixed